src/OpenTelemetry.Api/README.md

```diff
@@ -19,13 +18,7 @@
   Shim](#instrumenting-a-libraryapplication-with-opentelemetryapi-shim)
 * [References](#references)
 
-## Installation
```

Comment:
Do you want to remove the installation section and bury it in the introduction section?

Reply:
The installation is about installing the Otel.Api package. It may or may not be needed - if one is using DiagnosticSource alone, then Otel.Api is not required right? So pushed it elsewhere.

Reply:
added back for consisteny with all the other docs.